lib/CXGN/BrAPI/v2/Lists.pm

```diff
@@ -308,6 +340,18 @@ sub store {
 			});
 			my $reference_result = $references->store();
 		}
+		# Store additional Info
+		if( $additional_info_hash_ref ) {
+			my $dbh = $self->bcs_schema->storage()->dbh();
+			my $type_id = SGN::Model::Cvterm->get_cvterm_row($self->bcs_schema, 'list_additional_info', 'list_properties')->cvterm_id();
```

Comment:
I think this fetch could be moved outside of the for-loop

Reply:
Thanks for the catch.
Done